### PR TITLE
feat: also support lowercase matchespattern function

### PR DESCRIPTION
**Pull request by johannes-vogel**

### CODE REVIEW

File: db-service/lib/cql-functions.js [hunk 1]
```diff
@@ -99,6 +99,13 @@ const StandardFunctions = {
    * @returns {string}
    */
   matchesPattern: (x, y) => `(${x} regexp ${y})`,
+  /**
+   * Generates SQL statement that matches the given string against a regular expression
+   * @param {string} x
+   * @param {string} y
+   * @returns {string}
+   */
+  matchespattern: (x, y) => `(${x} regexp ${y})`,
```
Comment (BobdenOs) [inline db-service/lib/cql-functions.js:108]: This function is not defined in the other databases ?

Comment (johannes-vogel) [reply db-service/lib/cql-functions.js:108]: Isn't this the generic cql functions that we introduce?

Comment (BobdenOs) [reply db-service/lib/cql-functions.js:108]: Yes, but these are the database specific translations into SQL. I checked and it seems that this currently only works for sqlite.

So this PR should add a test using the function and the posters and HANA versions.

Comment (johannes-vogel) [reply db-service/lib/cql-functions.js:108]: Got it, will do.

**Review by BobdenOs — APPROVED** [review]

**Pull request merged**